doc/source/cluster/cluster_under_construction/ray-clusters-on-vms/user-guides/configuring-autoscaling.rst

```diff
@@ -2,55 +2,44 @@
 
 .. _deployment-guide-autoscaler-under-construction:
 
-Autoscaling with Ray
---------------------
+Configuring the Ray Autoscaler
+==============================
 
-Ray is designed to support highly elastic workloads which are most efficient on
-an autoscaling cluster. At a high level, the autoscaler attempts to
-launch/terminate nodes in order to ensure that workloads have sufficient
-resources to run, while minimizing the idle resources.
+This page provides an overview of the parameters for the Ray autoscaler. A Ray cluster by default runs an autoscaler that modifies
+the number of nodes based on the resources requested by the task, actor, or placement group. If there is insufficient resource to serve
```

Comment:
vs. the key concepts page, "placement group" can be kept here because we're going into a bit more depth on how the autoscaler works here

Reply:
"is insufficient resource" -> "are insufficient resources"

Reply:
sg